tests/containers/skopeo.pm

```diff
@@ -0,0 +1,119 @@
+# SUSE's openQA tests
+#
+# Copyright 2023-2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: skopeo
+# Summary: Test basic skopeo commands.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';    # used in select_serial_terminal
+use utils 'zypper_call';    # used in zypper_call
+use version_utils 'package_version_cmp';
+use version_utils 'is_transactional';
+use transactional;
+
+sub run {
+    my ($self, $args) = @_;
+
+    # Ensure skopeo is installed
+    # people make a $packages array which is populated with the packages that
+    # they want installed.
+
+
+    ######### Set required variables
+
+    # Required packages
+    my $packages = 'skopeo jq';    # and add whatever else should be missing with push function
+
+    select_serial_terminal();    # Select most suitable text console
+
+    # Set a variable for my remote image
+    my $remote_image = 'docker://registry.suse.com/bci/bci-busybox:latest';
+    # Set a variable for my working directory
+    my $workdir = '/tmp/test';
+
+    ######### Ensure pre-requisites
+
+    # Install required packages
+    # Make logic to account for SLE-Micro environment with transactional-update
+    is_transactional ? trup_install($packages) : zypper_call("--quiet in $packages", timeout => 120);
+
+    # Skopeo version emits a reasonable output
+    # Verify that skopeo version is greater or equal to 1.12.0.
+    die('wrong skopeo version') unless (package_version_cmp('1.12.0', script_output('skopeo --version | cut -d" " -f3')) <= 0);
+    # skopeo --version | cut -d" " -f3 --> separates "skopeo version 1.12.0" output into and array and takes third value "1.12.0"
+
+    # Create test directory
+    script_run("mkdir /tmp/test");
+
+    ######### Inspect a docker repository
```

Comment:
Maybe instead, the use of `record_info();` can be implemented to help future inspection in case of a failure. 🤔

Reply:
Done!